lib/components/app/app.css

```diff
@@ -72,21 +72,24 @@
   display: flex;
   justify-content: center;
 }
-.view-switcher button.btn-link {
+.view-switcher a {
   color: rgba(255, 255, 255, 0.85);
   border-radius: 15px;
+  font-size: 14px;
+  padding: 6px 12px;
+  user-select: none;
 }
-.view-switcher button.btn-link.active {
+.view-switcher a.active {
   background: rgba(0, 0, 0, 0.15);
   font-weight: 600;
 }
-.view-switcher button.btn-link:hover,
-.view-switcher button.btn-link:focus {
+.view-switcher a:hover,
+.view-switcher a:focus {
```

Comment:
Can we add a `cursor: pointer` here? It doesn't seem to be happening automatically all the time on Firefox.

Reply:
Good observation! Fixed in https://github.com/opentripplanner/otp-react-redux/pull/1183/commits/f5db10fddfe45ecbad66f9b5ef781e62341ce698.